lambdanotes/src/components/notecard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './notecard.css';
+
+const NoteCard = (props) => {
+  const previewLength = 150;
+  let preview = "";
+  if (props.note.content.length > previewLength) preview = props.note.content.substr(0, previewLength - 4) + " ...";
```

Comment:
Excellent! Like we talked about, if you want to have your ellipses to click for more on _each_ card preview, consider limiting the number of line breaks as well.